packages/ketcher-core/src/application/render/raphaelRender.js

```diff
@@ -96,6 +96,17 @@ Render.prototype.page2obj = function (event) {
   return this.view2obj(pp)
 }
 
+/**
+ * @param {Vec2} coord Vector in Raphael coordinate system,
+ *                     scaled atom's position with offset
+ * @returns {Vec2} Vector in View coordinate system,
+ *                 coordinate relative to viewport (excluding toolbar)
+ */
+Render.prototype.raphael2View = function (coord) {
```

Comment:
naming for 'raphael2View' is not clear, from my side it looks like it does not reflect the reason and the meaning. Also for me, it is hard to recognize from implementation - what is the difference with the .obj2view(vec)? 
Could you clarify?

Reply:
Maybe `raphael2View` is not a good name, I'll rename it in [#2697](https://github.com/epam/ketcher/issues/2697) if needed. As for something like `obj2View`, `page2Obj`, etc. I don't have an accurate idea of what they are, but I guess they are different coordinate systems, like

Reply:
@yuleicul thanks for the thorough analysis. 
But my point is - if we merge it and release it, our users will be able to use this method ()it is a public API of ketcher-core functionality, isn't it? And later if we will try to rename it we will have to do a deprecation process. 
Probably for now, it would be better to hide this ability somehow from the public to be able to refactor without issues.